hyperjumptech/monika
# feat: add support for postman config file

# Monika PR
Fix #273

## What issue this PR is trying to solve
Currently the configuration is only based on monika.json specific config. Need to add capability to read configuration from Postman file.

## How this PR solve the issue
Add capability to run probe read from Postman file definition.

## How to test
1. run monika with `--postman` flag set with the path to the postman json file 
2. It should be running well to monitor url set from HAR file
